File: c#/desafio.cs
using System;

public class Desafio {

    public static void Main() {

        float valorSalario = float.Parse(Console.ReadLine());
        float valorBeneficios = float.Parse(Console.ReadLine());

        float valorImposto = 0;
        if (valorSalario >= 0 && valorSalario <= 1100) {
            valorImposto = 0.05 * valorSalario;

        } else if ( valorSalario >= 1100.01 && valorSalario <=2500) {
            valorImposto = 0.10 * valorSalario;

        } else {
            valorImposto = 0.15 * valorSalario;
        }

        float saida = valorSalario - valorImposto + valorBeneficios;
        Console.WriteLine(saida.ToString("0.00"));
    }
}

File: python/desafio.py
def calcular_imposto(salario):
    aliquota = 0.00
    if (salario >= 0 and salario <= 1100):
        aliquota = 00.05
    elif (salario >= 1100.01 and salario <= 2500):
        aliquota = 00.10  
    else:
        aliquota = 00.15  
    
    return aliquota * salario

valor_salario = float(input())
valor_beneficios = float(input())

valor_imposto = calcular_imposto(valor_salario)

saida = valor_salario - valor_imposto + valor_beneficios
print(f'{saida:2f}')